001-xoxoxo-obj/harness.py

```diff
@@ -22,13 +22,13 @@ def Start(self):
 
       while True:
         move = self._inputs[player_id].get_move()
-        if move is None:
+        if move is False:
           self._output.show_move_error(player)
-          continue 
+          continue
 
         if self._game.make_move(move) is False:
           self._output.show_move_error(player)
-          continue
+          break

```

Comment:
Nie rozumiem natomiast tej zmiany - spowoduje ona w zasadzie tyle, że ponownie będzie wyrysowana plansza i będzie pokazane, którego gracza tura to jest. Raczej zostawiłbym tu continue żeby zachować zachowanie w zgodzie z tym sprzed kilku linii.